dev/integration_tests/android_views/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-all.zip
```

Comment:
Re rerun the migrator on all these files. Current code had 7.4.2 this is 7.4. I have a comment suggesting we use 7.6.1.

Blocking comment.